packages/website/src/search/StandaloneSearchBar.tsx

```diff
@@ -1,120 +1,30 @@
-        </Button>
-    );
-
-    const SuggestionListBox = () => {
-        const results: IItemBoxProps[] = state.suggestions.map((result) => ({
-            value: result.rawValue,
-            displayValue: result.highlightedValue,
-        }));
-
-        return (
-            <>
-                {focused && (state.suggestions.length > 0 || state.isLoadingSuggestions) && (
-                    <ListBox
-                        classes={['search-results-container sentence-case']}
-                        isLoading={state.isLoadingSuggestions}
-                        items={results}
-                        onOptionClick={(item) => {
-                            controller.selectSuggestion(item.value);
-                            navigateToResultPage(item.value);
-                        }}
-                    />
-                )}
-            </>
-        );
-    };
+const StandaloneSearchBar = () => {
+    const engine = useContext(EngineContext);
 
     return (
         <div className="plasmaSearchBar">
             <form autoComplete="off" onSubmit={(event) => event.preventDefault()}>
-                <input
-                    className="search-bar"
-                    type="search"
-                    placeholder={'Find a component...'}
-                    value={state.value}
-                    onChange={(event) => controller.updateText(event.target.value)}
-                    onKeyDown={(event) => {
-                        if (event.key === 'Enter') {
-                            navigateToResultPage(state.value);
-                        } else if (event.key === 'Escape') {
-                            controller.clear();
-                            (event.target as HTMLInputElement).blur();
-                        }
-                    }}
-                    onFocus={() => setFocused(true)}
-                    onBlur={() => setFocused(false)}
-                />
-                <ClearButton />
-                <SearchButton />
-                {/* 
-                    To toggle the feature flag, copy and paste those commands in the dev tool console: *
-                        
-                        localStorage.setItem('query-suggestions', true) to show the query suggestions *
-                        localStorage.setItem('query-suggestions', false) to hide the query suggestions *
-
-                    You need to reload the page for it to take effect.  
-                */}
-                {showQuerySuggesttions ? <SuggestionListBox /> : null}
+                <AtomicSearchInterface
+                    engine={engine}
+                    reflectStateInUrl={false}
+                    // eslint-disable-next-line @typescript-eslint/no-empty-function
+                    onReady={async () => {}} // noop
+                >
+                    <AtomicSearchBox
+                        numberOfQueries={5}
+                        redirectionUrl={'/Search'} // always redirect to real website
```

Comment:
I think something like this would do, if you want to redirect to the current site instead of the real one

```tsx
import {useRouter} from 'next/router';

const router = useRouter();

redirectionUrl={`${router.basePath}/Search`}
```